engine/src/main/java/org/terasology/world/internal/EntityAwareWorldProvider.java

```diff
@@ -118,14 +116,14 @@ public void shutdown() {
     @Override
     public Block setBlock(Vector3ic pos, Block type) {
         if (GameThread.isCurrentThread()) {
-            EntityRef blockEntity = getBlockEntityAt(JomlUtil.from(pos));
+            EntityRef blockEntity = getBlockEntityAt(pos);
             Block oldType = super.setBlock(pos, type);
             final Set<Class<? extends Component>> retainComponents =
                     Optional.ofNullable(blockEntity.getComponent(RetainComponentsComponent.class))
                             .map(retainComponentsComponent -> retainComponentsComponent.components)
                             .orElse(Collections.emptySet());
             if (oldType != null) {
-                updateBlockEntity(blockEntity, JomlUtil.from(pos), oldType, type, false, retainComponents);
+                updateBlockEntity(blockEntity, pos, oldType, type, false, retainComponents);
```

Comment:
a bunch of removing `JomlUtil.from`, which makes me feel like it's moving in the right direction :+1: